Automattic/wp-calypso
# Separate out refund policy for new domain registrations and renewals

#### Changes proposed in this Pull Request

This change implements separate wording for the domain refund policy that applies based on whether the domain is a new registration or a renewal, instead of language that applies to both situations.

#### Testing instructions

Using both the existing and new checkout flows, get to the main checkout UI for the following flows (you don't need to complete the purchase):
* Test buying a new domain with a plan; the checkout terms should include the following text:
   * You understand that [domain name refunds](https://wordpress.com/support/manage-purchases/#refund-policy) are limited to 96 hours after registration.
* Test buying a new domain as part of a plan purchase (i.e. via NUX); the checkout terms should include the following text:
   * You understand that [domain name refunds](https://wordpress.com/support/manage-purchases/#refund-policy) are limited to 96 hours after registration. Refunds of paid plans will deduct the standard cost of any domain name registered within a plan.
* Test renewing a domain; the checkout terms should include the following text:
   * Please note: to receive a [refund for a domain renewal](https://wordpress.com/support/manage-purchases/#refund-policy), you must [cancel your domain](https://wordpress.com/support/domains/#cancel-a-domain-name) within 96 hours of the renewal transaction. Canceling the domain means it will be deleted and you may not be able to recover it.

In each case, verify that the links work and link to the appropriate documentation.

_Note:_ You should be able to trigger the composite checkout by adding `?flags=composite-checkout-force` to your URL.

**Note:** I believe we don't support renewals and new registrations in the same cart at present, but the logic ought to show both messages in cases where both types of purchase are present.

<img width="719" alt="updated_domain_renewal_terms" src="https://user-images.githubusercontent.com/3376401/83045420-d3833380-a045-11ea-9891-cd49c6379e66.png">
